maxim/logger/writer.py

```
@@ -366,24 +405,79 @@ def flush_log_files(self):
+                    len(files),
+                    MAX_REPLAY_FILES_PER_CYCLE,
+                )
+                files = files[:MAX_REPLAY_FILES_PER_CYCLE]
             for file in files:
-                with open(os.path.join(self.logs_dir, file), "r") as f:
-                    logs = f.read()
+                filepath = os.path.join(self.logs_dir, file)
+                try:
+                    with open(filepath, "r") as f:
+                        logs = f.read()
+                except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)